### PR TITLE
Update HTTP specification

### PR DESCRIPTION
**Pull request by loan21xw**
## Describe your changes

Update HTTP documentation and update transaction API

## This change belongs to the following squad(s)
- Database
- [X] HTTP
- OCPP

## I have created unit tests for my code
- [X] Yes
- No
- No, I need help with this

## Checklist before creating my pull request
- [X] My code is properly commented where needed
- [X] I have updated the documentation with my changes/additions
- [X] I have reviewed my code before creating this pull request
- [X] I have made sure that I am targeting the correct branch

### CODE REVIEW

File: backend-app/src/presentation-layer/app.js [hunk 1]
```diff
@@ -35,7 +35,8 @@ module.exports = function({chargersRouter, transactionsRouter, authenticationRou
 
     app.use('/swagger', swaggerUi.serve, swaggerUi.setup(openApiDocument));
     app.use('/chargers', chargersRouter)
-    app.use('/transactions', transactionsRouter)
+    app.use('/transaction', transactionRouter)
+    app.use('/transactions', transactionRouter)
```
Comment (SweOlle) [inline backend-app/src/presentation-layer/app.js:39]: Should this still be here?

File: backend-app/src/presentation-layer/transaction-router-api.js [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+            if (!transactionID) { response.status(500).json(errors); return; }
+
+            data.transactionID = transactionID;
+            response.status(201).json(data);
+        });
+    });
+
+    router.put("/start/:transactionID", verifyUser, function (request, response) {
+        const transactionID = request.params.transactionID;
+        console.log(1);
```
Comment (SweOlle) [inline backend-app/src/presentation-layer/transaction-router-api.js:71]: Remove logging?

File: backend-app/src/presentation-layer/app.js (hunk 1 above)
Comment (loan21xw) [reply backend-app/src/presentation-layer/app.js:39]: Yes, this is backwards compatibility. We might also do different responses depending on the URL if the Mobile teams wants that.

File: backend-app/src/presentation-layer/transaction-router-api.js [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+    }
+
+    const router = express.Router();
+
+    router.post("/", verifyUser, function (request, response) {
+
+        console.log("Entering Post transactions");
+        const { connectorID, paymentType } = request.body;
+
+        const data = {
```
Comment (SweOlle) [inline backend-app/src/presentation-layer/transaction-router-api.js:31]: Mobile squads want the response to look like this:
{
      "klarnaConsumerToken": "",
      "transactionID": ""
}

**Review by SweOlle — APPROVED** [review]
Looks good!

**Pull request merged**